src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -7,13 +7,19 @@ import Link, { LinkProps } from '../Link';
 export interface BreadcrumbItem {
   /** The URL that this Breadcrumbs should navigate to when clicked */
   href: string;
-
-  /** The text that should be displayed on this particular Breadcrumb */
-  text: string;
+  /** Title is the text that should be displayed on this particular Breadcrumb
+   * element can be: numbers, strings, elements or an array (or fragment) containing these types.
+   */
+  title: React.ReactNode;
+  /**
+   * @deprecated use node property instead.
```

Comment:
This commentt is  wrong, since `node` doesn't exist

Reply:
sorry renamed to title!